scripts/deploy-proxy-server-as-service.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+#########################
+# Create Detination Dir #
+#########################
+
+DESTINATION=/usr/lib/ranger/ranger-gcs-plugin-proxy-server
```

Comment:
This is assuming Ranger is installed under /usr/lib/ranger, right? Is this always true?